src/chainparams.cpp

```diff
@@ -553,6 +572,12 @@ class CRegTestParams : public CChainParams {
         consensus.vUpgrades[idx].nActivationHeight = nActivationHeight;
     }
 
+    void UpdateFundingStreamParameters(Consensus::FundingStreamIndex idx, Consensus::FundingStream fs)
+    {
+        assert(idx > Consensus::FIRST_FUNDING_STREAM && idx < Consensus::MAX_FUNDING_STREAMS);
```

Comment:
Should this be `>= Consensus::FIRST_FUNDING_STREAM`?

Reply:
Yes; that's a copy-paste error from the NU logic (where we don't allow Sprout to be altered).